app/views/layouts/application.html.erb

```diff
@@ -14,8 +14,46 @@
     <%= turbo_refreshes_with method: :morph, scroll: :preserve %>
     <%= content_for :head %>
   </head>
```

Comment:
I deleted the layout/messages template and elevated most of that into layouts/application.

But there is still a messages/left_column and messages/right_column